.pipelines/e2e.yml

```diff
@@ -86,14 +111,20 @@ jobs:
           run_selenium
           validate_selenium_running
 
+          update_role_sets
+
           run_mimo_actuator
           validate_mimo_actuator_running
 
+          update_role_sets
+
           run_rp
           validate_rp_running
 
           register_sub
 
+          # cluster is cleaned up in a later step. Keep it around for now to enable must-gather collection on error
+          export E2E_DELETE_CLUSTER=false
```

Comment:
Should we note or create a task to change it to true whenever we are done with collecting errors.

Reply:
The comment is referring to the e2e cluster, not the variable.
If this job (`Execute Tests`) fails, the pipeline tries to collect a must gather in the subsequent job. Regardless of weather the tests failed or succeeded, the cleanup job is always run.
This means that we should always keep the cluster around after the e2e tests finished (by exporting `E2E_DELETE_CLUSTER=false`), as it's cleaned up anyway. And if we don't, then the pipeline won't be able to collect the must gather in case of e2e test failure.

Does that make sense?